theme/sass/components/_footer.scss

```diff
@@ -65,52 +65,27 @@
 
             h2 {
                 color: $accent-color-one;
+                padding: 0;
+                margin-bottom: 5px;
             }
 
-            .social-media {
+            .social-icons {
                 display: flex;
                 flex-direction: row;
                 padding: 0;
                 list-style-type: none;
+                justify-content: center;
 
-                .twitter {
-                    margin-right: 3px;
+                .fa-facebook-f,
+                .fa-twitter,
+                .fa-instagram {
+                    color: white;
```

Comment:
Should this not be using a variable?

Reply:
oh yes, so sorry I forgot about this one.